### ISSUE TITLE
Entwine build fails for LAS file from cloud storage

### ISSUE THREAD

**Issue by fletsch** [connormanning/entwine#328]
Hi, first of all thanks for creating this great library!

I recently ran into an issue when building an EPT from a LAS file which is stored on Azure Blob Storage, similiar to issue #294.
Running this (`entwine version: 3.1.1`)
```bash
export AZURE_STORAGE_ACCOUNT=some_storage_account_name
export AZURE_STORAGE_ACCESS_KEY=some_acees_key
entwine build -i az://some_container/some_las_file.las -o .
```
will fail with 
```bash
Encountered an error: [json.exception.out_of_range.403] key 'schema' not found
Exiting.
```
Running the build with the exact same file locally, does work. And interestingly, the issue does not happen for compressed LAZ files. 

I tried to find the root of this error and saw that it is caused by this line: https://github.com/connormanning/entwine/blob/2a0524bd74074ed9a1852f51c410ed4d1cf81939/entwine/util/info.cpp#L94
which throws the error
```bash
readers.las: Invalid point count: 9917348. Number of points too large for file size.
```
So it seems the pdal las reader cannot handle the las file with zero points but a header indicating a number of points which was generated [here](https://github.com/connormanning/entwine/blob/2a0524bd74074ed9a1852f51c410ed4d1cf81939/entwine/util/io.cpp#L194). The corresponding code in pdal is [here](https://github.com/PDAL/PDAL/blob/3334e6ecd666b2ff8bc7c427bf7864a3e8ba64f1/io/LasReader.cpp#L333). However, I could not find out why exactly pdal is validating the file size also for a preview (and especially why for a pointless las files but not for pointless laz files)

A temporary workaround is appending the `--deep` option for all builds from las files.

Any help is highly appreciated!

**Comment by connormanning** [comment]
Can you provide the file?  Depending on the LAS version, I wonder if it's something like that we are setting the `point count` field but forgetting about the `legacy point count` which is getting picked up (or the reverse or something similar).  So the file would be helpful to see what's going wrong exactly and to test a fix.  Feel free to email it to me if it's not a publicly available file.

**Comment by fletsch** [comment]
Yes sure, this happend with publicly available files from [swissSURFACE3D](https://www.swisstopo.admin.ch/en/height-model-swisssurface3d). I tested it e.g. with [this file](https://data.geo.admin.ch/ch.swisstopo.swisssurface3d/swisssurface3d_2020_2660-1266/swisssurface3d_2020_2660-1266_2056_5728.las.zip), but it should be reproducible with any of the files I think.

**Comment by xavierfontaine** [comment]
I face a similar issue, but with S3 instead of Azure, and the missing key is `bounds` instead of `schema`.

The OP's workaround (to use `--deep`) adds an overhead but works well.

My example file is [this one](https://drive.proton.me/urls/2RJA2N5WK8#sOdKUeA3CJuF).

My setting:
* pdal: 2.8.4
* entwine: 3.1.0
